crates/ruff_python_parser/tests/snapshots/invalid_syntax@expressions__named__missing_expression_0.py.snap

```diff
@@ -0,0 +1,43 @@
+---
+source: crates/ruff_python_parser/tests/fixtures.rs
+input_file: crates/ruff_python_parser/resources/invalid/expressions/named/missing_expression_0.py
+---
+## AST
+
+```
+Module(
+    ModModule {
+        range: 0..75,
+        body: [
+            Expr(
+                StmtExpr {
+                    range: 71..72,
+                    value: Name(
+                        ExprName {
+                            range: 71..72,
+                            id: "x",
+                            ctx: Load,
+                        },
+                    ),
+                },
+            ),
+        ],
+    },
+)
+```
+## Errors
+
+  |
+1 | # There are no parentheses, so this isn't parsed as named expression.
```

Comment:
Is there a reason why we don't parse this as a named expression? Can we document if there's a reason that we don't parse this as a named expression or is it just because that's what we do today?

Reply:
It is because the top level grammar for expression is `star_expressions` which doesn't allow named expression. That's what we use. If we would allow named expression in this case, then I think there would be two error messages (1) expected expression after `:=` and (2) unparenthesized named expression cannot be used here. Maybe this can be looked into when working on error recovery.